legal-api/report-templates/ceaseReceiver.html

```diff
@@ -0,0 +1,41 @@
+[[macros.html]]
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <title>Cease Receiver</title>
+      <meta charset="UTF-8">
+      <meta name="author" content="BC Registries and Online Services">
+      [[common/style.html]]
+   </head>
+   <body>
+      <div class="header">
+         <table class="header-table" role="presentation">
+            <tr class="no-page-break">
+               <td>
+                  [[logo.html]]
+               </td>
+               <td>
+                  <div class="report-type">
+                     CEASE RECEIVER
+                  </div>
+                  <div class="report-type-desc">{{ entityDescription }} - {{ entityAct }}</div>
+               </td>
+            </tr>
+         </table>
+         <div class="business-name-header">
+            <label class="lbl-business-name">
+               {% if business.legalName %}
+                  {{ business.legalName }}
+               {% elif nameRequest.legalName %}
+                  {{ nameRequest.legalName }}
+               {% else %}
+                  {{numberedDescription}}
+               {% endif %}
```

Comment:
I don't think these else conditions are required. Its not possible to change the legalName in ceaseReceiver filing